srcpkgs/lightdm-slick-greeter/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'lightdm-slick-greeter'
+pkgname=lightdm-slick-greeter
+version=2.0.9
+revision=1
+build_style=meson
+hostmakedepends="pkg-config gettext vala"
+makedepends="gtk+3-devel lightdm-devel libcanberra-devel ayatana-ido-devel
+ xapps-devel"
+depends="hicolor-icon-theme"
+conf_files="/etc/lightdm/slick-greeter.conf"
+short_desc="Light Display Manager GTK+ Greeter from Linux Mint"
+maintainer="Emil Tomczyk <emru@emru.xyz>"
+license="GPL-3.0-only"
+homepage="https://github.com/linuxmint/slick-greeter"
+distfiles="https://github.com/linuxmint/slick-greeter/archive/refs/tags/${version}.tar.gz"
+checksum=fa0146862ac0967a1a333f9b553d60a5c625c99b903d01aefe9b87bfdb111c29
+provides="lightdm-greeter-1_0"
+
+post_install() {
+	vmkdir etc/lightdm
+	vinstall ${FILESDIR}/slick-greeter.conf 0644 etc/lightdm
```

Comment:
instead of vendoring this, we can grab it with sed: `sed -n -e 's/^\s*//' -e '/\[Greeter\]/,/^$/p' README.md > "$DESTDIR/etc/lightdm/slick-greeter.conf"`